src/interfaces/business-interfaces.ts

```diff
@@ -66,7 +66,8 @@ export interface ApiBusinessIF {
   startDate: ApiDateTimeUtc
   submitter?: string // not used
   taxId?: string // aka Business Number // may be absent
-  warnings: Array<BusinessWarningIF>
+  warnings: Array<BusinessWarningIF>,
+  operatingName?: string
```

Comment:
Looks good but:
- interface fields don't need commas separating them
- please keep this list in alphabetical order (to make it easier for the next dev to find things)